logging/structured/src/main/scala/tofu/logging/Logging.scala

```diff
@@ -19,50 +20,68 @@ import scala.reflect.ClassTag
   */
 trait LoggingBase[F[_]] {
 
-  /** push new message to log, level will be automatically checked
+  /** Push new message to log, level will be automatically checked
+    *
     * @param level desired level of logging, message will not be rendered\sent, if logging level
     *              of current logger is not low enough
     * @param message composed string with `{}` placeholders for values
     *                 do not create strings on each call, use constant template string instead
     * @param values  log parameters , values of types having `Loggable` instance would be converted automatically
     */
-  def write(level: Level, message: String, values: LoggedValue*): F[Unit]
+  def write(level: Level, location: Option[Location], message: String, values: LoggedValue*): F[Unit]
```

Comment:
This change is not binary compatible.
It would be better if old methods kept the same